### PR TITLE
Update documentation for SSO and IdP integration

### PR DESCRIPTION
**Pull request by jfairbank**
This PR updates the "About Identity Management" page with steps to test the SSO login process and adds a FAQs section that answers some common questions about Anypoint's IdP integration.

The FAQs were borne out of conversations with our QA team who felt that our documentation does not make it clear what happens in certain scenarios with regard to IdP integration.

Thanks to @samjonester for pairing on these doc updates!

**NOTE:** this PR is dependent on #1623 and should be merged after it.

### CODE REVIEW

File: access-management/v/latest/external-identity.adoc [hunk 1]
```diff
@@ -32,6 +32,32 @@ After you’ve made your selection, you are ready to set up your Anypoint Platfo
...
+
+== FAQs
+
+*Q: What happens when a user logs into the configured IdP for the first time?*
+
+A: An Anypoint user is created in your organization using the user's IdP information, including username, name, and email.
+
+*Q: What happens when a user updates their profile information in the configured IdP?*
+
+A: After logging into Anypoint via the IdP, the user's profile information will be updated within Anypoint.
```
Comment (Schoonology) [inline access-management/v/latest/external-identity.adoc]: This should probably be `The next time that user logs into Anypoint via...`.

Comment (krishahn) [inline access-management/v/latest/external-identity.adoc]: ok, tks

**Comment by krishahn** [comment]
working on it (just to let my co-writers know that i got it covered)

**Comment by krishahn** [comment]
Thanks for the contribution, guys. In Dynamic Client Registration, step 4 includes examples for entering URLs for Okta and OpenAM. In Manual Client Registration, step 6 includes examples for URLs those two providers plus PingFederate. PingFederate s/b included in the Dynamic Client Registration, right (just forgotten)? If so, don't worry about correcting your PR. I'm making changes to conform to the Docs style guide.

**Comment by krishahn** [comment]
Thanks for the contribution, guys. Is there any reason for the inconsistency in the base urls for the PF examples? For the Authorize and Token URLs it's https://example.com:9031/as, but for the User Info URL it's https://example.com:9031/idp/? If it's ok, I'll just make the base URLs consistent. No need for another PR from you. I'm making other changes to conform to the Docs style guide.

**Comment by jfairbank** [comment]
@krishahn The URL difference is intentional and should remain.

**Comment by krishahn** [comment]
close per Ashley

**Pull request closed without merging**